### DISCUSSION TITLE
Allow "pinning" apps so that they appear on every workspace

### DISCUSSION THREAD

**Discussion by axtloss** [YaLTeR/niri#286] [Ideas]
My workflow includes having qmmp on the far left of my workspace so that i can always slide it in and control the music, on desktop environments like gnome i would make the window visible on all desktops, so that i can always have quick access to the window no matter what workspace i am on.

This could be done either through a keybinding to toggle it or a window rule to activate it for specific windows (or both)

This may be similiar (or the same) as #149, but I dont really understand what the sticky columns exactly mean in that case.

**Comment by bobblkabb** [comment]
Think that only makes sense if there is a floating window function.

**Reply by axtloss** [reply]
How so? I still want the window to be part of the workspace sheet so that it scrolls with it and everything, just that its visible on every workspace

**Reply by YaLTeR** [reply]
Oh I see

**Comment by YaLTeR** [comment]
Yeah, I think the floating layer will work similarly to what you describe.

**Reply by axtloss** [reply]
It wouldn't, with the floating layer the window wouldn't be part of the workspace sheet anymore
